src/EnergyPlus/FluidProperties.cc

```diff
@@ -3046,7 +3030,7 @@ namespace FluidProperties {
         if (!state.dataGlobal->WarmupFlag && ErrorFlag) {
             ++this->errors[(int)RefrigError::SatTempDensity].count;
             // send warning
-            if (this->errors[(int)RefrigError::SatTempDensity].count <= df->RefrigErrorLimitTest) {
+            if (this->errors[(int)RefrigError::SatTempDensity].count <= state.dataFluidProps->RefrigErrorLimitTest) {
```

Comment:
[src/EnergyPlus/FluidProperties.cc:2998]:(style),[variableScope],The scope of the variable 'df' can be reduced.